crates/wasmtime/src/sig_registry.rs

```diff
@@ -14,20 +13,23 @@ use wasmtime_environ::{ir, wasm::WasmFuncType};
 #[derive(Debug, Default)]
 pub struct SignatureRegistry {
     wasm2index: HashMap<WasmFuncType, VMSharedSignatureIndex>,
+    index_map: HashMap<VMSharedSignatureIndex, Entry>,
+}
```

Comment:
Since this now stores trampolines, it would be great to update the comment.

Reply:
Indeed!